src/api/app/assets/images/request_icon.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Please add the license information we have talked about

Reply:
We already had a `LICENCE.md` inside `app/assets/images/icons`, so we have moved the icon to that sub-directory and added the text in that existing file.